cache_utils/decorators.py

```diff
@@ -102,10 +114,19 @@ def wrapper(*args, **kwargs):
             if value is None:
                 logger.debug("Cache MISS: %s" % key)
                 value = func(*args, **kwargs)
-                cache_backend.set(key, value, timeout, **backend_kwargs)
+
+                value_to_persist = (
+                    NONE_SENTINEL
+                    if value is None
+                    else value
+                )
+
+                cache_backend.set(key, value_to_persist, timeout, **backend_kwargs)
                 logger.debug("Cache SET: %s" % key)
             else:
                 logger.debug("Cache HIT: %s" % key)
+                if isinstance(value, NoneSentinel):
```

Comment:
👍  awesome, this should avoid issues with unpickling it later for redis and other backends that use pickle.